### DISCUSSION TITLE
Feature Request - Option to open maximized as default

### DISCUSSION THREAD

**Discussion by o-l-a-v** [File-New-Project/EarTrumpet#674] [Ideas]
I always expand/ maximize EarTrumpet when using it, would be awesome with an option to open maximized/ expanded as default.

Default (as of now)
![image](https://user-images.githubusercontent.com/6450056/49147687-98dff400-f305-11e8-954a-1b6f80e6fbf4.png)

Default (When feature request is implemented, and manually enabled in EarTrumpet settings)
![image](https://user-images.githubusercontent.com/6450056/49147663-8f568c00-f305-11e8-93f9-3d1e2aadd6b0.png)

**Comment by yani** [comment]
This would be amazing if dragging and dropping were implemented. :)

**Comment by SunFlea** [comment]
This would be very nice since I use VoiceMeeter as my default device but have to adjust my speakers to actually change the volume.

**Comment by Tester798** [comment]
Maybe saving the state of last window between relaunches would work?
Then probably we don't need the option in settings in this case.

I can implement it, but not sure which option is better.
